CMakeLists.txt

```diff
@@ -86,9 +86,20 @@ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 if (MAC_TABLET_ENABLE)
     add_compile_definitions(__mac_tablet__=1)
 endif ()
+
+find_package(OpenSSL REQUIRED)
+find_library(BRPC_LIBRARY brpc)
+find_library(PROTOBUF_LIBRARY protobuf)
+find_library(GLOG_LIBRARY glog)
+find_library(GFLAGS_LIBRARY gflags)
+find_library(UNWIND_LIBRARY unwind)
+find_library(LEVELDB_LIBRARY leveldb)
+find_library(Z_LIBRARY z)
```

Comment:
might be other place as well: 
https://github.com/4paradigm/OpenMLDB/blob/aac8167ca47b47871f5c6902abedeb6a7d294c43/src/CMakeLists.txt#L123